packages/react-devtools-scheduling-profiler/src/view-base/utils/scrollState.js

```diff
@@ -18,10 +18,10 @@ import {clamp} from './clamp';
  *     |<-------------------length------------------->|
  * ```
  */
-export type ScrollState = $ReadOnly<{|
```

Comment:
why remove the ReadOnly?

Reply:
Because I'm relying on the state being _mutable_ now (since it's stored and restored in context). I _could_ recreated it but then this would also require React to schedule context updates and re-render and since this is mostly in the imperative world (outside of React) anyway, that felt unnecessary.

I guess it could go either way.